src/test/java/org/takes/http/BkParallelTest.java

```diff
@@ -23,15 +23,32 @@
  */
 package org.takes.http;
 
+import java.net.Socket;
+import java.util.concurrent.ExecutorService;
+import org.junit.Test;
+import org.mockito.Mockito;
+
 /**
  * Test case for {@link BkParallel}.
  *
  * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
  * @version $Id$
  * @since 0.15.2
- * @todo #220:30min/DEV This unit test not implemented but has to be.
- *  E.g. you could pass your own a ExecutorService implementation to the ctor
- *  and verify execute() calls.  But feel free to try another way.
  */
-public class BkParallelTest {
+public final class BkParallelTest {
+    /**
+     * BkParallel can call execute() when called accept().
+     */
+    @Test
+    public void callsExecuteOnAccept() {
+        final ExecutorService executor = Mockito.mock(ExecutorService.class);
+        final BkParallel parallel = new BkParallel(
+            Mockito.mock(Back.class),
+            executor
+        );
+        parallel.accept(Mockito.mock(Socket.class));
+        Mockito.verify(executor, Mockito.times(1)).execute(
+            Mockito.<Runnable>any()
+        );
```

Comment:
@cyberone it should be used to synchronize between the 2 threads (the test thread and the thread created with the call to `BkParallel.accept`. If we just run a verify with no latch, it will probably fail most of the time since the newly created thread won't have run yet. With the latch, the test thread will call `accept` then wait on the latch until the new thread calls `latch.countdown`. Of course, the wait should not be forever - a 1 minute timeout is reasonable when calling `latch.await`.